cs-engine/install.md

```diff
@@ -63,27 +63,31 @@ to update its RHEL kernel.
     $ sudo yum install docker-engine
     ```
 
-6.  Enable the Docker daemon as a service and start it.
+6.  Configure devicemapper:
+
+    By default, the `devicemapper` graph driver does not come pre-configured in a production ready state. Follow the documented step by step instructions to [configure devicemapper with direct-lvm for production](../../engine/userguide/storagedriver/device-mapper-driver/#/for-a-direct-lvm-mode-configuration) in order to achieve the best performance and reliability for your environment.
```

Comment:
/ping @cpuguy83 or @mavenugo for a validation that this config is really needed for a production deployment.